src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Alert coverage gaps
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+  - Alert coverage gaps
+metaDescription: Learn how alert coverage gaps in New Relic help you identify gaps in your alert system. 
+---
+import alertcoveragegaps from 'images/alert-coverage-gaps-homepage.png'
+
+import addanalert from 'images/add-an-alert.png'
+
+import createapolicyalertcoveragegaps from 'images/create-a-policy-coverage-gaps.png'
+
+Alert coverage gaps highlight areas in your stack where you're missing observability coverage. In our Alerts and AI section [we've created a homepage](https://one.newrelic.com/nrai/detection-gaps/home) where you can see all your uncovered entities so you can assess what information you're team is lacking and address the knowledge gap. 
```

Comment:
I would say `where you're missing alerts coverage`

Reply:
Also, there is a typo.  Should be `your` team.  Not `you're` team.

Reply:
I would say `address the coverage gap` rather than `address the knowledge gap`.

Reply:
Thanks! I fixed this.